apypie/api.py

```diff
@@ -189,6 +189,8 @@ def http_call(self, http_method, path, params=None, headers=None, options=None,
 
         request = self._session.request(http_method, full_path, **kwargs)
         request.raise_for_status()
+        if request.status_code == 204:  # no content
+            return {}
```

Comment:
Why an empty dict instead of None? I think this is typically the response for a DELETE, but where else?

Reply:
I have not tried None. I guess that is equally fine.